transports/small-webrtc-transport/src/smallWebRTCTransport.ts

```diff
@@ -9,18 +9,38 @@ import {
 import { MediaManager } from "../../../lib/media-mgmt/mediaManager";
 import { DailyMediaManager } from "../../../lib/media-mgmt/dailyMediaManager";
 
-const SIGNALLING_TYPE = "signalling";
-
-enum SignallingMessage {
-  RENEGOTIATE = "renegotiate",
+class TrackStatusMessage {
+  type = "trackStatus";
+  receiver_index: number;
+  enabled: boolean;
+  constructor(receiver_index: number, enabled: boolean) {
+    this.receiver_index = receiver_index;
+    this.enabled = enabled;
+  }
 }
 
+const RENEGOTIATE_TYPE = "renegotiate" as const;
```

Comment:
curious, why the `as const`?

Reply:
We don't actually need it, probably just a bad code completion from my IDE.